### PR TITLE
DOCS-2221: TC Updating/organizing LDAP

### PR DESCRIPTION
**Pull request by BonnieF**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Comment by Mrt134** [comment]
@beanpole135 I added you as a reviewer for whenever Bonnie is ready. She'll remove the WIP label.

**Comment by BonnieF** [comment]
I am waiting for a review from the TrueCommand team to confirm the updated information is accurate.

File: static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part
Comment (beanpole135) [inline static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part:21]:
```suggestion
| | Example: "ldap.mycorp.com:636" (SSL port is typically 636 for AD/LDAP)|
```

File: static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part
Comment (beanpole135) [inline static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part:46]:
```suggestion
* Domain: "dc=mycorp,dc=com"
* User ID Field: "uid" 
```

File: static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part
Comment (beanpole135) [inline static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part:62]:
```suggestion
* Domain: "dc=mycorp,dc=com"
* User ID Field: "uid"
* BIND User Domain: "uid=binduser,cn=read-only-bind,dc=mycorp,dc=com"
* BIND Password: "pre-shared-key"
```

File: static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part
Comment (beanpole135) [inline static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part:71]:
```suggestion
* (Option 1) The custom certificate authority must be registered with TrueCommand via the Certificates Tab of the Administrator Settings.
```

**Review by beanpole135 — CHANGES_REQUESTED** [review]
Cleanup some of the formatting.
Since you replaced the API-specific terminology with the UI terminology, you need to ensure you do that same conversion everywhere else too. I fixed a couple of them, but there are some more places which reference API fields.
The table also looks a bit funny with the example as a second table row instead of within the same table cell. You should be able to multiple multiple lines of text within a single table cell by wrapping the contents within an HTML element like `<span>text..<br>on line 2</span>`.

**Comment by BonnieF** [comment]
I asked @beanpole135  to check for accuracy. I need to sync with @Mrt134  about an edit choice.

File: static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part
Comment (beanpole135) [inline static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part:31]:
```suggestion
TrueCommand supports two common methods of validating LDAP user credentials:
```

File: static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part
Comment (beanpole135) [inline static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part:33]:
```suggestion
#### Direct Bind
```

File: static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part
Comment (beanpole135) [inline static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part:43]:
```suggestion
#### Indirect Bind
```

File: static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part
Comment (beanpole135) [inline static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part:59]: Put a little warning box here stating that SSL connections are *required* for AD/LDAP authentication.

**Review by beanpole135 — CHANGES_REQUESTED** [review]

File: static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part
Comment (BonnieF) [reply static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part:59]: I have added the warning

**Review by beanpole135 — APPROVED** [review]

**Review by tonyriv3 — APPROVED** [review]

**Pull request merged**
